channel.go

```diff
@@ -304,9 +311,20 @@ func NewChannel(serviceName string, opts *ChannelOptions) (*Channel, error) {
 	}
 	ch.peers = newRootPeerList(ch, opts.OnPeerStatusChanged).newChild()
 
-	if opts.Handler != nil {
+	switch {
+	case len(opts.SkipHandlerMethods) > 0 && opts.Handler != nil:
+		sm, err := toServiceMethodSet(opts.SkipHandlerMethods)
+		if err != nil {
+			return nil, err
```

Comment:
can you add coverage for this by trying to construct a channel with an invalid method in `SkipHandlerMethods`?